sources/Trigger+ReplaceTrigger_m.h

```diff
@@ -0,0 +1,12 @@
+//
+//  Trigger+ReplaceTrigger_m.h
```

Comment:
Did you mean to include this file?

Reply:
no… I guess this was a result of refactoring in XCode… 😕will remove it